api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -229,6 +231,41 @@
     public void fireEvent(Object event, Annotation... qualifiers);
 
     /**
+     * Fire an asynchronous event and notify observers.
```

Comment:
Do we need these methods on BeanManager as well? I know that fireEvent() is already duplicated since CDI 1.0 though I am wondering if we want to follow this. The BeanManager already has way too many methods. In addition, the CDI 1.0 fireEvent() method makes sense on BeanManager since it allows events to be fired from extensions (where Event built-in bean cannot be injected). With fireAsync() there may not be the need.